root/static/scripts/tests/release-editor/fields.js

```diff
@@ -183,3 +183,55 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
     );
     t.ok(_.last(medium.tracks()).isDataTrack());
 });
+
+fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
+    t.plan(8);
+    
+    var lengths = [
+        "5",
+        "69",
+        "174",
+        "6000",
+        "7400",
+        "10000",
+        "96900",
+        "3723494",
+    ];
+
+    var output = [
+        "0:05",
+        "1:09",
+        "2:54",
+        "1:00:00",
+        "1:14:00",
+        "1:00:00",
+        "10:09:00",
+        "?:??",
+    ];
+
+    var medium = new fields.Medium({ tracks: [ {} ] }, release);
+
+    medium.tracks()[0].formattedLengthChanged(lengths[0]);
```

Comment:
But don't we want to know the output instead of display what is expected in the message? I mean sure it works, but I would like to know the output, for now, maybe change it after a final commit, some fundamental errors in the field.js messing it up.